app/lib/item.rb

```diff
@@ -0,0 +1,16 @@
+# => Don't touch this class
+require_relative 'delegate_items'
+
+class Item
+  attr_accessor :name, :sell_in, :quality
+
+  def initialize(name, sell_in, quality)
+    @name = name
+    @sell_in = sell_in
+    @quality = quality
+  end
+
+  def to_s()
+    "#{@name}, #{@sell_in}, #{@quality}"
```

Comment:
same inconvenient here, you could call the methods for addressing those instance variables instead of accessing them directly.